README.md

```diff
@@ -52,11 +52,6 @@ cp .env.template .env
 ```
 2. Fill the required environment variables in the `.env` file.
 3. Source the `.env` file.
-```bash
```

Comment:
why this is not required? You are not using .env file now?

Reply:
`load_dotenv()` added in client.py will take care of this

Reply:
`load_dotenv()` added in client.py will take care of this